lib/util/json-writer/editor-config.spec.ts

```diff
@@ -0,0 +1,67 @@
+import mock from 'mock-fs';
```

Comment:
I'm unsure about this new dependency. @JamieMagee @zharinov WDYT?

Reply:
Yeah, we normally include a `__fixtures__` directory, and put test fixtures in there. Separates out the tests from and sample data.